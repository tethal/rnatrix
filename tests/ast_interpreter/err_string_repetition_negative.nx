fun main() {
    print("hello" * -3);
}