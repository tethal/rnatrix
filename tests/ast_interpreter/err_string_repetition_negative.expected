<string>:2:19: error: string repetition count cannot be negative
    print("hello" * -3);
                  ^
